Gemfile

```
@@ -30,6 +30,9 @@ gem "jsonapi-serializer", "~> 2.2.0"
 # Catch unsafe migrations in development
 gem "strong_migrations", "~> 1.4.4"
 
+# Use Money-Rails
+gem "money-rails", "~>1.12"
```

What do you think of using the latest version of this [gem](https://rubygems.org/gems/money-rails)? After updating you might need to run `bundle install` to update the dependencies.
```suggestion
gem "money-rails", "~> 1.15.0"
```